packages/ckeditor5-emoji/tests/emojimention.js

```diff
@@ -133,6 +134,24 @@ describe( 'EmojiMention', () => {
 			editorElement.remove();
 		} );
 
+		it( 'should not modify mention feed when emoji plugin is already in mention feeds', async () => {
+			const editorElement = document.createElement( 'div' );
+			document.body.appendChild( editorElement );
+
+			const editor = await ClassicTestEditor.create( editorElement, {
+				plugins: [ EmojiMention, Mention ]
+			} );
```

Comment:
Creating a test editor in this way causes that real `EmojiDatabase` is loaded, so real `fetch()` call to CDN is performed.